components/app.js

```diff
@@ -4,11 +4,18 @@ class App {
     }
 
     handleGetGradesSuccess(grades) {
-        this.gradeTable.updateGrades(grades);               
+        this.gradeTable.updateGrades(grades); 
+        var average = 0;
+        var totalGrade = 0;
+        for(var i=0; i< grades.length ;i++)  
+            totalGrade += grades[i].grade;
+        average = totalGrade/3;

```

Comment:
This will only work when there are three grades, can you make it more generic so that it will support any number of grades?